_docs/style/focus-states.md

```diff
@@ -34,358 +41,454 @@ Focus is received by interactive components when the user clicks on them.  Focus
 
 <section class="static-section" markdown="1">
 
-## Composition
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Specifications
+
+Use the following specifications when you construct a UI element that is in focus.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
 
 <div class="hxRow" markdown="1">
+
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-Two visual features make up the focus indicator:
+### Composition
+
+Focus consists of the following elements:
 
-1. **The Glow** is always applied to the focused element.
-2. **The Border** is only applied to elements that already have a border.
+- **Glow (required)**: The glow is the ring that appears around the focused element.
```

Comment:
maybe call it a "hazy ring"? to distinguish it from the solid border line

Reply:
Good one, and done!